source/guide/cordova-plugins.md

```diff
@@ -147,3 +147,61 @@ export default {
 }
 </script>
 ```
+
+### Example: Device
+First step is to read the documentation of the Cordova plugin that we want to use. We look at [Cordova Plugins list](https://cordova.apache.org/docs/en/latest/#plugin-apis) and click on [Device doc page](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-device/index.html).
+
+This plugin initializes a global variable called `device` which describes the device's hardware and software. As Quasar is the one who captures this event, you need modify the .eslintrc.js file, leaving the property globals in this way:
+
+```
+globals: {
+    'ga': true, // Google Analytics
+    'cordova': true,
+    'device': true,
+    '__statics': true
+  }
+```
+
+We read the instructions on how to install this plugin. It's always a Cordova command. **So we "cd" into `/src-cordova`** (which is a Cordova generated folder) **and issue the install command form there**:
+```bash
+# from /src-cordova:
+$ cordova plugin add cordova-plugin-device
+```
+
+Now let's put this plugin to some good use. If you need the information of your device when starting the application, you will have to capture the created event. In one of your Quasar project's pages/layouts/components Vue file, we write:
+
+```html
+// some Vue file
+// remember this is simply an example;
+// only look at how we use the API described in the plugin's page;
+// the rest of things here are of no importance
```

Comment:
I would either leave out this commentary or put some real vue-esque cruft in here.